Tests/SwiftAlgorithmsTests/TestUtilities.swift

```diff
@@ -62,7 +63,30 @@ func XCTAssertEqualSequences<S1: Sequence, S2: Sequence>(
   _ message: @autoclosure () -> String = "",
   file: StaticString = #file, line: UInt = #line
 ) rethrows where S1.Element == S2.Element {
-  try XCTAssert(expression1().elementsEqual(expression2(), by: areEquivalent), message(), file: file, line: line)
+
+  func fail(_ reason: String) {
+    XCTFail(message().isEmpty ? reason : "\(message()) - \(reason)",
+            file: file, line: line)
+  }
```

Comment:
This looks good! Just to be on the safe side, let's store the message in its own variable first (inside the `fail` function) to avoid calling it twice.